docs/hub/models-faq.md

```diff
@@ -42,4 +42,10 @@ If the model card includes a link to a paper on arXiv, the Hugging Face Hub will
 <img class="hidden dark:block" width="300" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/datasets-arxiv-dark.png"/>
 </div>
 
-Read more about paper pages [here](./paper-pages).
\ No newline at end of file
+Read more about paper pages [here](./paper-pages).
+
+## How are download stats generated for models?
```

Comment:
let's define an anchor so that we can link directly to here (w/o risking breaking it with a header change)

Or even have this section as its own doc?